kafka-bin-scripts/README.adoc

```diff
@@ -94,6 +95,7 @@ To enable the Kafka scripts to access a Kafka instance, you must configure the c
 .Prerequisites
 ifndef::qs[]
 * You have the generated credentials for your service account. You copied this information previously for the Kafka instance in {product} by selecting the options menu (three vertical dots), clicking *Connection*, and creating the service account.
```

Comment:
I've always wondered -- are we assuming that the user who is following non-Getting-Started quick starts is the same user who created the instance -- and thus the service account and its generated credentials? That's how it reads, so I'm assuming yes?

Reply:
Separate but maybe related comment to @jciroli-rh's 🤔 

This statement seems to imply that a service account that you create from the drawer is somehow tied to a Kafka instance. That wasn't really the case at summit. But going forward, there's only a relationship between the two through the process of defining access on the Kafka instance. 
> You copied this information previously for the Kafka instance

To @jciroli-rh's comment, there are at least 2 places, soon to be 3, where you can create a service account:
* the Service Accounts page
* the Connection drawer for a Kafka instance
* eventually the Connection drawer for a Service Registry instance

Reply:
@jciroli-rh and @jgiardino You're both right here. We've had to minimize cross-refs in our quick starts in the UI and isolate cross-refs instead in "Additional resources" as much as possible, because we don't yet have a good way of cross-reffing in-UI. Especially since prereqs should and will be shown in UI eventually. This is highly impractical most of the time, but one way we've gotten around it is by at least jogging the user's memory like this. But again, you're correct that the direction is getting outdated with new UI features. Also not always the case that the user created these things from the UI to begin with, so better to poise it as just an option or instruction to do, regardless of how they did it the first time.

So how about this rewrite for both prereqs. It's more detailed than we typically go with prereqs, but to be super clear on what to do, and to avoid sending them on a link chase (and we're asked to avoid linking anyway), this is a compromise. Wdyt?

> * You have the generated credentials for your service account. You copied this information previously for the Kafka instance in {product}. To regenerate the credentials, open the Application Services web console and use the *Service Accounts* page to find your service account and update the credentials.
> * You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, open the {product} web console, select your Kafka instance, and use the *Access* tab of the Kafka instance to find your service account permission settings.

Then for the quick start procedures that require just bootstrap server endpoint, like later in bin scripts and in Kcat:

>* You have the bootstrap server endpoint for your service account. You copied this information previously for the Kafka instance in {product}. To relocate the server endpoint, open the {product} web console, select your Kafka instance, select the options menu (three vertical dots), and click *Connection*.
>* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, open the {product} web console, select your Kafka instance, and use the *Access* tab of the Kafka instance to find your service account permission settings.


And then for quick start procedures that require multiple or all of the above, like Quarkus:

>* You have the bootstrap server endpoint and the generated credentials for your service account. You copied this information previously for the Kafka instance in {product}. 
To relocate the server endpoint, open the {product} web console, select your Kafka instance, select the options menu (three vertical dots), and click *Connection*.
To regenerate the credentials, open the Application Services web console and use the *Service Accounts* page to find your service account and update the credentials.

>* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, open the {product} web console, select your Kafka instance, and use the *Access* tab of the Kafka instance to find your service account permission settings.

Reply:
Ugh, yeah, sorry. It's correct in the docs, just old copy-paste. Fixed now in my previous comment.

Reply:
The context in this thread has been super helpful -- thank you both, Stetson and Jenn. If you have what you need here, we can resolve the comment.